aptos-move/framework/aptos-framework/doc/aptos_coin.md

```diff
@@ -4,7 +4,7 @@
 # Module `0x1::aptos_coin`
 
 This module defines a minimal and generic Coin and Balance.
-modified from https://github.com/move-language/move/tree/main/language/documentation/tutorial
+modified from https://github.com/vgao1996/move/tree/main/language/documentation/tutorial
```

Comment:
@vgao1996 , this can't be right, right? :-)

Reply:
This is indeed unintentional -- probably the artifacts of a rebase or search & replace). Thanks for catching it!

Reply:
Fixed